cli/data.go

```diff
@@ -565,6 +623,74 @@ func (c *viamClient) deleteBinaryData(filter *datapb.Filter) error {
 	return nil
 }
 
+func (c *viamClient) addTagsToBinary(filter *datapb.Filter, tags []string) error {
```

Comment:
Maybe specifying here somehow that this calls the ByFilter API. Originally, when I read this, I thought that the logic for both adding tags by filter and by ID would be handled here.